src/pages/common.js

```diff
@@ -0,0 +1,47 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Can you rename this file to `common.styles.js` due to our style guide.

Reply:
fixed